### ISSUE TITLE
obtain deleted mail folders via gosdk delta call

### ISSUE THREAD

**Issue by luigi-fast** [microsoftgraph/msgraph-sdk-go#586]
Hello,
I need to obtain the delta of a user mail folders but I need to filter only deleted one.   According to your api documentation, it can be achieve like this with a GET call:
GET /me/mailfolders/{id}/messages/delta?changeType=deleted

I've tried it with postman and it works.

My question is: how can I do that with the gosdk?  The only query parameter I can use is the ones we found is the users.ItemMailFoldersDeltaRequestBuilderGetQueryParameters

**Comment by baywet** [comment]
Hi @luigi-fast ,
Thanks for using the go SDK and for reporting this.
@rkodev this query parameter is missing from the metadata. We should add an entry to the xslt to insert the missing query parameter.
Irvine should be able to help you with this if you need more specific directions.

**Comment by luigi-fast** [comment]
Yes, I would appreciate a workaround while waiting for the update.

Thanks

**Comment by luigi-fast** [comment]
@irvinesunday I don't know if my last post was clear but I would like some advice to achieve what I want to obtain: delta of deleted mail folders with the gosdk.  Maybe it would be possible for the moment with a raw url or something like that?

Thanks

**Comment by knirb** [comment]
This is a blocker for us in a client project. Would be greatly appreciated if this got some focus. Our only option at the moment  is to convert this to the raw request, and not very pleasant.

**Comment by luigi-fast** [comment]
I'm still waiting for a workaround.  Nothing to say about our problem?

**Comment by rkodev** [comment]
@luigi-fast Thanks for your patience, This issue should be fixed in the latest version v1.32.0 of this sdk. Please confirm this code works

```go

	changeType := "deleted"
	configuration := &users.ItemMailFoldersItemMessagesDeltaRequestBuilderGetRequestConfiguration{
		Headers: nil,
		QueryParameters: &users.ItemMailFoldersItemMessagesDeltaRequestBuilderGetQueryParameters{
			ChangeType: &changeType,
		},
	}
	response, err := client.Me().MailFolders().ByMailFolderId("inbox").Messages().Delta().Get(context.Background(), configuration)
	if err != nil {
		return
	}
```

**Comment by luigi-fast** [comment]
@rkodev Sorry for the delay.  I've tried it and it works well.  Thanks.
